<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Narrator</class>
 <widget class="QWidget" name="Narrator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>790</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>790</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gaming Narrator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../assets/logo.png</normaloff>../assets/logo.png</iconset>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label_bg">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/a/final.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/assests/assests/final.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="b_settings">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>310</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight SemiConde</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#b_settings{
	background-color: rgba(29, 53, 87, 225);
	color:rgba(241, 250, 238);
	border-radius: 3px;
}
QPushButton#b_settings:hover{
	background-color: rgba(29, 53, 87, 128)
}</string>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_tutorial">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>260</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight SemiConde</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#b_tutorial{
	background-color: rgba(29, 53, 87, 225);
	color:rgb(241, 250, 238);
	border-radius: 5px;
}

QPushButton#b_tutorial:hover{
	background-color: rgba(29, 53, 87, 128)
}
</string>
    </property>
    <property name="text">
     <string>Tutorial</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_start">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>540</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight SemiConde</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#b_start{
	background-color: rgba(168, 218, 220, 128);
	color:rgba(241, 250, 238);
	border-radius: 3px;
}

QPushButton#b_start:hover{
	background-color: rgba(168, 255, 220, 255)
}</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_stop">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>540</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight SemiConde</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#b_stop{
	background-color: rgba(230, 57, 70, 128);
	color:rgb(241, 250, 238);
	border-radius: 3px;
}
QPushButton#b_stop:hover{
	background-color: rgba(230, 57, 70, 255)
}</string>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>170</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight SemiConde</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(241, 250, 238);</string>
    </property>
    <property name="text">
     <string>Gaming Narrator</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="qt.qrc"/>
 </resources>
 <connections/>
</ui>
